ts-service/src/index.ts

```diff
@@ -1,36 +1,44 @@
-import yupana from "@yupana/core-lib-poc";
-// import { notify } from "./utils";
-
-// const yupana = require("@yupana/core-lib-poc");1
-
-function notify(message: string, ...objs: any[]) {
-    const baseMsg = "FROM TS-SERVICE: ";
-
-    console.log(`${baseMsg}${message}`);
-
-    objs.forEach(obj => {
-        console.log(JSON.stringify(obj, null, 2));
-    });
+import {Service} from "@yupana/framework-service-types";
+import {YContext} from "@yupana/framework-types/lib/YContext";
+import {YRole} from "@yupana/framework-types/lib/YRoles";

```

Comment:
I missed in the original PR, why do we have Yroles and YSessions (plural) ?

Reply:
Will update the library in v0.1.5